app/assets/stylesheets/components/_nav.scss

```
@@ -10,8 +12,19 @@
 }
 
 .sidenav-mobile .usa-nav__close {
-  @include add-background-svg('close-blue-60v-alt');
+  @include add-background-svg('close-primary');
   @include u-square(6);
   background-position: center center;
   background-repeat: no-repeat;
 }
+
+@media (prefers-reduced-motion) {
+  // The design system should be responsible for disabling inessential animations when user prefers
+  // reduced motion. This is not currently implemented, but the styles is implemented ad hoc here
```

singular/plural disagreement? "style is" or "styles are" ?
```suggestion
  // reduced motion. This is not currently implemented, but the style is implemented ad hoc here
```

---

> singular/plural disagreement? "style is" or "styles are" ?

Yeah, good catch.